### ISSUE TITLE
Make creating an album easier with an "Add Album" button

### ISSUE THREAD

**Issue by Wimboson** [SimpleMobileTools/Simple-Gallery#2137]
Thanks for this great app. I am almost completely satisfied with it.
However, it would be nice if creating albums could be easier.

It would be great to have a plus symbol in the bottom right when selecting photos.
![Screenshot_20210518-091653](https://user-images.githubusercontent.com/84132993/118620074-7720a300-b7c5-11eb-8d4f-4a122ab1b4d7.png)

If you press the plus symbol, an overview of the albums opens. 
here you can either select an existing album or press + to create a new one.
![Screenshot_20210518-094350](https://user-images.githubusercontent.com/84132993/118620131-8273ce80-b7c5-11eb-8667-146f9a7e3b01.png)

Maybe in the settings could be an option to specify a standard path for albums. 

In my opinion it would be a nice little simplification of the handling.

**Comment by AlbatorV** [comment]
Good idea but add "Copy to" and "Move to" items when I tap + button before display destination screen
